ddl-scripts/create_tables_postgres.sql

```diff
@@ -23,6 +23,22 @@ CREATE TABLE IF NOT EXISTS event_journal(
 
 CREATE INDEX IF NOT EXISTS event_journal_slice_idx ON event_journal(slice, entity_type_hint, db_timestamp);
 
+CREATE TABLE IF NOT EXISTS snapshot(
+  slice INT NOT NULL,
```

Comment:
yes, good to include the `slice` to have the same kind of primary key even though we don't have any plan to use it for snapshots

Reply:
That was what I was thinking, for sharding to work as well in a distributed db (without having to have different DAO impl)